pkgs/development/tools/build-managers/alibuild/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, python}:
+
+python.pkgs.buildPythonPackage rec {
```

Comment:
If this is a standalone python application you should use  buildPythonApplication.